_config.yml

```diff
@@ -160,7 +160,12 @@ component_releases:
 
 release_archive:
     flink:
-      - version_short: "1.12"
+      -
+        version_short: "1.11"
```

Comment:
All the 1.11 series should be grouped together.

Reply:
True. I thought they were ordered by release date. Will fix this. Thanks.